### ISSUE TITLE
client_sandbox path setup is not behaving as expected

### ISSUE THREAD

**Issue by AymenFJA** [radical-cybertools/radical.pilot#3072]
When setting up as the following: 
```python
rp.Session(cfg={"client_sandbox": "new_path"})
```
It is totally ignored. And the session path created in `cwd`.

If we do:
```python
import radical.pilot as rp

s = rp.Session(cfg={'base': 'runinfo'})
new session: [rp.session.controlplane.kc-internal.019656.0007]                 \
zmq proxy  : [tcp://172.30.1.2:10001]  
```
The expected behavior is for the session to be in `runinfo`. We get an unusual behavior (2 sessions are created in nested folders and another `runifno` is created under ```runinfo/rp.session.xxxxx/runifno/rp.session.xxx```):
```
In [11]: ls rp.session.controlplane.kc-internal.019656.0007/
proxy_control_pubsub.log  proxy_control_pubsub.prof  proxy_state_pubsub.log  proxy_state_pubsub.prof  proxy_task_queue.log  proxy_task_queue.prof  radical.pilot.bridge.log

In [12]: ls runinfo/rp.session.controlplane.kc-internal.019656.0007/
cmgr.0000.log         heartbeat_pubsub.prof  radical.saga.log                                          runinfo/          stager_request_queue.err     state_pubsub.err
cmgr.0000.prof        log_pubsub.err         radical.utils.log                                         server.0000.log   stager_request_queue.out     state_pubsub.out
control_pubsub.err    log_pubsub.out         rp.session.controlplane.kc-internal.019656.0007.log       server.0000.prof  stager_request_queue.prof    state_pubsub.prof
control_pubsub.out    log_pubsub.prof        rp.session.controlplane.kc-internal.019656.0007.prof      stager.0000.err   stager_response_pubsub.err
control_pubsub.prof   radical.saga.api.log   rp.session.controlplane.kc-internal.019656.0007.reg.log   stager.0000.out   stager_response_pubsub.out
heartbeat_pubsub.log  radical.saga.cpi.log   rp.session.controlplane.kc-internal.019656.0007.reg.prof  stager.0000.prof  stager_response_pubsub.prof
```

**Comment by mtitov** [comment]
Corresponding paths should be updated for `Proxy` component

https://github.com/radical-cybertools/radical.pilot/blob/a5e65328fd884a4fb4ddb792b346d5f94c23577d/src/radical/pilot/proxy.py#L232-L258

**Comment by AymenFJA** [comment]
This is addressed by: #3073, and it should be closed once the PR (already approved) is merged.
